core/node/api_server/src/tx_sender/result.rs

```diff
@@ -69,6 +69,8 @@ pub enum SubmitTxError {
     Internal(#[from] anyhow::Error),
     #[error("transaction failed block.timestamp assertion")]
     FailedBlockTimestampAssertion,
+    #[error("sender address {0} is not in allow list")]
+    SenderNotInAllowList(Address),
```

Comment:
Bikeshedding: Wouldn't the caller have the initiator address readily available as a part of the submitted transaction? So this info looks redundant. If it's retained, it's weird that the called contract address isn't included as well.